test/app.listen.js

```diff
@@ -15,4 +16,18 @@ describe('app.listen()', function(){
       done();
     });
   })
+
+  it('should callback on HTTP server errors', function(done){
+    var app = express();
+    var app2 = express();
+
+    var server = app.listen(9999, function(err){
```

Comment:
Yeah, I can make this change for sure.  Let me know if you also want me to move this PR to the 5.0 branch and I will do it all at once.